wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -0,0 +1,284 @@
+
+### Gameplay and scoring
+
+#### Se faccio uno score su lazer, verrà visualizzato sul mio profilo?
+
+gli score verranno mostrati sotto la sezione "Partite Recenti" ma non in "Migliore Performance" per ora.
+
+#### Se imposto un punteggio su lazer, darà punti performance?
+
+I punteggi avranno gia' i punti performance calcolati (puoi vederlo nella sezione "Partite recenti" del tuo profilo), ma non contribuiranno ancora al valore totale.
+
+#### Lazer usa ScoreV2?
+
+Lazer attualmente utilizza un nuovo sistema di punteggio sperimentale che e' simile a ScoreV2 ma non e' identico. questo e' ancora sotto sviluppo e stiamo raccogliendo pareri dai player in base a quanto bene va in vari contesti (solo play, classifiche, tornei etc.)
+
+<!-- lint ignore no-heading-punctuation -->
+
+#### preferisco il vecchio sistema di punteggio, dove il punteggio diventa veramente grande.
+
+Puoi effettivamente modificare l'impostazione `Modalità di visualizzazione del punteggio` in `Classico` per ripristinare lo stile esplosivo del punteggio in tutto il gioco! Non sarà un abbinamento perfetto, ma ti darà la stessa sensazione del punteggio classico e verrà applicato ovunque ti aspetteresti che sia.
+
+#### Se imposto un punteggio su lazer, rimarrà per sempre?
+
+Anche se cercheremo di preservare il maggior numero possibile di punteggi, **non garantiamo che i punteggi rimarranno a tempo indeterminato**. In qualsiasi momento potremmo decidere di cancellare un sottoinsieme o tutti i punteggi per preservare l'equilibrio del gioco.
+
+#### I punteggi impostati su stable saranno visualizzati in lazer?
+
+Sì. Una volta terminato il bilanciamento della combinazione di punteggi in lazer e in stable, entrambi saranno visibili.
+
+#### Tutte le mod saranno Classificato?
+
+Per ora, i punteggi di tutte le combinazioni di mod appaiono nelle classifiche. Si sta ancora discutendo se i punteggi daranno punti alle prestazioni con tutte le mod (e se lo faranno, se ci sarà un bonus o una penalità).
+
+#### Non mi piacciono le nuove meccaniche di gioco. Posso ripristinare le vecchie meccaniche di gioco come su stable?
+
+Prova ad applicare la mod "classic", che ripristinera' gran parte del vecchio comportamento a cui sei abituato. Assicurati anche di controllare le impostazioni offerte dalla mod classica, in quanto ti consentiranno di personalizzare ulteriormente la tua esperienza e di capire quali modifiche vengono applicate (in quanto sono tutte elencate).
+
+### Skinning e UI
+
+#### Qualcosa si comporta in modo diverso rispetto a stable e non mi piace!
+
+Eseguire la configurazione guidata nella parte superiore delle impostazioni e controllare le impostazioni nella schermata `Comportamento`. Qui sono elencate molte delle impostazioni comuni che sono state modificate per impostazione predefinita. C'è anche un singolo pulsante che si può premere per applicare i vecchi comportamenti come punto di partenza per il vostro viaggio con il lazer.
```

Comment:
we don't really know how "behaviour" is adapted in the italian version of the game so i'm not sure if "comportamento" is the right choice for now